src/test/java/org/killbill/billing/plugin/qualpay/EmbeddedDbHelper.java

```diff
@@ -30,24 +30,34 @@ public class EmbeddedDbHelper {
 
     private static final EmbeddedDbHelper INSTANCE = new EmbeddedDbHelper();
     private EmbeddedDB embeddedDB;
+    
+    private static final String DDL_FILE_NAME = "ddl.sql";
 
     public static EmbeddedDbHelper instance() {
         return INSTANCE;
     }
 
     public void startDb() throws Exception {
-        embeddedDB = PlatformDBTestingHelper.get().getInstance();
 
+        embeddedDB = PlatformDBTestingHelper.get().getInstance();
+        
         // Needed, otherwise get Caused by: java.sql.SQLException: No suitable driver found for jdbc:mysql:<connection-url>
         if (embeddedDB.getDBEngine().equals(DBEngine.MYSQL)) {
             Class.forName("com.mysql.cj.jdbc.Driver");
         }
-
+        
         embeddedDB.initialize();
         embeddedDB.start();
 
-        final String ddl = "ddl-" + embeddedDB.getDBEngine().name().toLowerCase() + ".sql";
-        embeddedDB.executeScript(TestUtils.toString(ddl));
+        final String databaseSpecificDDL = "ddl-" + embeddedDB.getDBEngine().name().toLowerCase() + ".sql";
+        try {
+            embeddedDB.executeScript(TestUtils.toString(databaseSpecificDDL));
+        } catch (final IllegalArgumentException e) {
+            // Ignore, no engine specific DDL
```

Comment:
Is this a legit case?

Reply:
I added this code to eliminate [this test failure](https://github.com/killbill/killbill-qualpay-plugin/actions/runs/3871365446/jobs/6599100516#step:7:1899). TBH, I just copy-pasted this code from other plugins. Most plugins (like [Stripe](https://github.com/killbill/killbill-stripe-plugin/blob/ec31abcfa6ef3a95a877ce05d6f9db370c17471d/src/test/java/org/killbill/billing/plugin/stripe/EmbeddedDbHelper.java#L47), [Avatax](https://github.com/killbill/killbill-avatax-plugin/blob/08a56cc65fbe01736b1b7931731b88f32a9e59ed/src/test/java/org/killbill/billing/plugin/avatax/EmbeddedDbHelper.java#L50)..) have similar code.  Maybe we could revisit this code for all plugins once the  plugins are released and stable?